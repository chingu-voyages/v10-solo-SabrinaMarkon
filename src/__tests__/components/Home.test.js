import React from 'react';
import { shallow } from 'enzyme';

// components
import Home from '../../components/Home';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Home />);
});

describe('Testing Home component', () => {
  it('should render an h1 heading with the correct text', () => {
    expect(wrapper.find('h1').text().toEqual('Home'));
  });
  it('it should match the Home snapshot', () => {
    // If snapshot exists, compare. If not, create the snapshot.
    expect(wrapper).toMatchSnapshot();
  });
});