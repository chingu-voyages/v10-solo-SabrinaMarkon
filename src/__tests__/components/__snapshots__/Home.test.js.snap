// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home component it should match the Home snapshot 1`] = `undefined`;
